About us, Login, Signup pages added

dc-30 opened the pull request:
This PR solves issues [#7](https://github.com/hkasera/WeCare/issues/7) and [#6](https://github.com/hkasera/WeCare/issues/6). We have attached screenshots of the same. Kindly review it. 
![screenshot from 2016-04-08 17_39_20](https://cloud.githubusercontent.com/assets/15409834/14389354/ed1edffa-fdcf-11e5-9873-b82462aa8bdd.png)
![screenshot from 2016-04-08 17_39_58](https://cloud.githubusercontent.com/assets/15409834/14389355/ed244fd0-fdcf-11e5-87a1-b55367056de9.png)
![screenshot from 2016-04-08 17_40_38](https://cloud.githubusercontent.com/assets/15409834/14389356/ed2732ae-fdcf-11e5-9133-2160960e44c3.png)
![screenshot from 2016-04-08 17_41_02](https://cloud.githubusercontent.com/assets/15409834/14389357/ed2c4a64-fdcf-11e5-8a03-2a88afa3583c.png)

hkasera commented on about.html:
```diff
@@ -0,0 +1,41 @@
+    <link rel="stylesheet" href="css/styleAbout.css" type="css/text">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=papyrus">
+    <title>We Care</title>
+    <style type="text/css">
```
Why css here and not in a separate file?

hkasera commented on about.html:
```diff
@@ -0,0 +1,41 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styleAbout.css" type="css/text">
+    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=papyrus">
```
Avoid loading fonts! You can read the pros and cons of this online. Unless necessary lets not use this.

hkasera commented on about.html:
```diff
@@ -0,0 +1,41 @@
+    </style>
+   
+</head>
+<body style="background-image:url('img/banner_new.jpg');
```
Inline styling is a bad idea. Make sure you look up these online and get back if you have any queries.

hkasera commented on login.html:
```diff
@@ -0,0 +1,43 @@
+  <title>Login</title>
+
+    <link rel="stylesheet" href="/css/style.css" media="screen" type="text/css" />
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Open+Sans:400,700">
```
Different fonts on different pages! We should maintain consistency.

hkasera commented on login.html:
```diff
@@ -0,0 +1,43 @@
+
+            <p><span class="fontawesome-user"></span><input type="text" value="Username" onBlur="if(this.value == '') this.value = 'Username'" onFocus="if(this.value == 'Username') this.value = ''" required></p> <!-- JS because of IE support; better: placeholder="Username" -->
+            <p><span class="fontawesome-lock"></span><input type="password"  value="Password" onBlur="if(this.value == '') this.value = 'Password'" onFocus="if(this.value == 'Password') this.value = ''" required></p> <!-- JS because of IE support; better: placeholder="Password" -->
+            <p><input type="submit" onclick="myFunction()" value="Sign In"></p>
```
Where is `myFunction` defined?

hkasera commented on public/css/style2.css:
```diff
@@ -0,0 +1,210 @@
+/*--
```
I don't like this naming convention. `style.css` and `style2.css` are not very informative names of the files.

hkasera commented on public/css/style.scss:
```diff
@@ -0,0 +1,137 @@
+/* ---------- http://fortawesome.github.com/Font-Awesome/ ---------- */
+/* ---------- http://weloveiconfonts.com/ ---------- */
+
+@import url(http://weloveiconfonts.com/api/?family=fontawesome);
```
Copying css is not a good idea if you don't know why you are using it! Please make sure you check if things are working cross browser!

hkasera commented on public/css/style.scss:
```diff
@@ -0,0 +1,137 @@
+/* ---------- ERIC MEYER'S RESET CSS ---------- */
+/* ---------- http://meyerweb.com/eric/tools/css/reset/ ---------- */
+
+@import url(http://meyerweb.com/eric/tools/css/reset/reset.css);
```
Do you know why you are using `reset.css`?

hkasera commented on public/css/styleAbout.css:
```diff
@@ -0,0 +1,42 @@
+
+}
+
+ body:after {
```
Why `body:after` and not `body`?

hkasera commented on server.js:
```diff
@@ -10,12 +10,16 @@ var moment = require('moment');
 
 
+
+
```
Avoid empty spaces!

hkasera commented on signup.html:
```diff
@@ -0,0 +1,32 @@
+<head>
+<title>Sign up</title>
+<meta charset="utf-8">
+<link href="css/style2.css" rel='stylesheet' type='text/css' />
```
Signup and Login should have common CSS considering they are quite similar pages.

hkasera commented on signup.html:
```diff
@@ -0,0 +1,32 @@
+<link href="css/style2.css" rel='stylesheet' type='text/css' />
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<script type="application/x-javascript"> addEventListener("load", function() { setTimeout(hideURLbar, 0); }, false); function hideURLbar(){ window.scrollTo(0,1); } </script>
+<link href='http://fonts.googleapis.com/css?family=Oxygen:400,300,700' rel='stylesheet' type='text/css'>
```
Inconsistency in font styles.

hkasera commented:
Please send wire frames or mock designs before implementing. Also, send which libraries you will use. This will avoid rework. Make designs consistent. Follow one convention and stick to it.

dc-30 commented:
We have created a single CSS file for signup and login. Also, we removed the inline styling. We have changed the files according to your remarks. Kindly review it.

dc-30 commented:
Attached are the screenshots for the same. 
![screenshot 14](https://cloud.githubusercontent.com/assets/15409834/14404725/1537a0f0-fe9b-11e5-9632-a91d5d1d12db.png)
![screenshot 15](https://cloud.githubusercontent.com/assets/15409834/14404727/1538172e-fe9b-11e5-9c31-4db823c33251.png)
![screenshot 16](https://cloud.githubusercontent.com/assets/15409834/14404726/1537dfca-fe9b-11e5-9cd3-238cc999c199.png)

hkasera commented on login.html:
```diff
@@ -0,0 +1,161 @@
+            
+            <div class="field-wrap">
+              <label>
+                Set A Password*
```
Make it _Password_

ppalod replied:
We changed it.

hkasera commented on login.html:
```diff
@@ -0,0 +1,161 @@
+<script src="/js/handlebars.js"></script>
+<script src="/js/jquery.easing.js"></script>
+<script src="/js/waterfall.js"></script>
+<script src="/js/index.js"></script>
```
What does this (**index.js**) file do?

ppalod replied:
index.js file used in login.html is to effect whether the sign up tab is active or login tab.

hkasera commented on login.html:
```diff
@@ -0,0 +1,161 @@
+<script src="/js/jquery.easing.js"></script>
+<script src="/js/waterfall.js"></script>
+<script src="/js/index.js"></script>
+<script src="/js/creative.js"></script>
```
What does this (**creative.js**) file do?

ppalod replied:
creative.js file used for implementing effects in HTML page For example :  smooth scrolling, fixed navbar,etc.
